.env.docker

```diff
@@ -5,3 +5,7 @@ DB_PORT=3306
 DB_DATABASE=laravel
 DB_USERNAME=user
 DB_PASSWORD=password
+KEY_HASH_MAC=token_password
```

Comment:
look at docker-compose.yml because we are setting some environment variables there